packages/component-base/src/element-mixin.d.ts

```diff
@@ -16,6 +17,6 @@ interface ElementMixinConstructor {
   finalize(): void;
 }
 
-interface ElementMixin extends DirMixin {}
+interface ElementMixin extends ReactiveControllerHost, DirMixin {}
```

Comment:
Extending `ReactiveControllerHost` as such adds `requestUpdate()` and `updateComplete` to all components' API definitions.

```suggestion
interface ElementMixin extends Pick<ReactiveControllerHost, 'addController' | 'removeController'>, DirMixin {}
```